packages/augur-core/source/contracts/libraries/token/ERC777.sol

```diff
@@ -222,7 +222,7 @@ contract ERC777 is IERC777, IERC20 {
     */
     function transferFrom(address holder, address recipient, uint256 amount) public returns (bool) {
         require(recipient != address(0), "ERC777: transfer to the zero address");
-        require(holder != address(0), "ERC777: transfer from the zero address");
+        require(holder != address(0));
```

Comment:
reason to remove the require messaging?

Reply:
To reduce the contract size so Universe deployment is below an acceptable gas limit :[